src/components/Header/Header.tsx

```diff
@@ -1,14 +1,22 @@
+import { useState } from 'react';
 import styles from './Header.module.css';
 import Logo from '../Logo';
 import NavBar from '../NavBar';
 import LogIn from '../LogIn';
+import Avatar from '../Avatar';
 
 function Header() {
+   const [isLogedIn, setIsLogedIn] = useState(true);
+
+   const handleLogOut = () => {
+      setIsLogedIn(!isLogedIn);
```

Comment:
```suggestion
      setIsLogedIn(prev => !prev);
```
A better practice as it makes sure you're using the most recent value of that state and not the state itself.

Reply:
noted.